pkg/controllers/nodeclaim/disruption/consolidation.go

```diff
@@ -76,8 +76,8 @@ func (c *Consolidation) Reconcile(ctx context.Context, nodePool *v1.NodePool, no
 	if !hasConsolidatableCondition {
 		log.FromContext(ctx).V(1).Info("marking consolidatable")
 
-		metrics.NodeClaimsDisruptedCounter.With(prometheus.Labels{
-			metrics.TypeLabel:     metrics.ConsolidationReason,
+		metrics.NodeClaimsDisruptedTotal.With(prometheus.Labels{
```

Comment:
We drop this, right? Along with the other disruption counters -- since we only disrupt the node when we are actually terminating the nodeclaim